src/BuildScriptGenerator/DefaultBuildScriptGenerator.cs

```diff
@@ -175,13 +184,31 @@ public bool TryGenerateBashScript(BuildScriptGeneratorContext context, out strin
             return resultPlatforms;
         }
 
-        private static string GetBenvArgs(Dictionary<string, string> benvArgsMap)
+        private static string GetBenvArgs(IDictionary<string, string> benvArgsMap)
         {
             var listOfBenvArgs = benvArgsMap.Select(t => $"{t.Key}={t.Value}");
             var benvArgs = string.Join(' ', listOfBenvArgs);
             return benvArgs;
         }
 
+        private void RunCheckers(
+            BuildScriptGeneratorContext ctx,
+            IDictionary<string, string> tools,
+            [NotNull] List<ICheckerMessage> checkerMessageSink)
+        {
+            var checkers = _checkers.WhereApplicable(tools);
+
+            using (var timedEvent = _logger.LogTimedEvent("RunCheckers"))
+            {
+                timedEvent.AddProperty(
+                    "checkersApplied",
+                    string.Join(',', checkers.Select(checker => checker.GetType().Name)));
```

Comment:
nit: `','` => `', '`

Reply:
this is for our own telemetries, though, NOT for displaying to customers

Reply:
I see...just though it will be easy to read...you can leave it